index.html

```diff
@@ -15,14 +15,20 @@
 <body>
     <header class="navbar-container">    
             <nav>
-                <a class="logo-item" href="index.html">John Doe</a>
+                <a class="logo-item" href="index.html">Peter chendi</a>
 
                 <img class="handberga" src="Images/Icons/Union.png" alt="handberga menu">
-            
+            <ul>
+                <li><a>Portfolio</a></li>
+                <li><a>About</a></li>
+                <li><a>Contact</a></li>
+                <li><a><img src="Images/Icons/Normalbutton/Tertiary/Icons/mail.png" alt="handberga menu"></a></li>
```

Comment:
The hamburger icon should not be visible in the desktop view. Kindly hide this when in desktop mode.
![image](https://user-images.githubusercontent.com/33268927/186835964-9ec11b6c-d685-4559-b609-fd994190cf9e.png)